### PR TITLE
Remove Bootstrap Property from Onboarding Docs

### PR DESCRIPTION
**Pull request by nagilson**
# Mostly Wholistic Summary
- Forgind made a PR where the bootstrap flag is set automatically when building which simplifies the dev process a lot.
- Some of the documentation wasn't changed when that got merged in and we didn't notice until onboarding a new teammate 

So, this PR does the following:
- Change all of the inferences to bootstrap except on bootstrapped_msbuild.sh files, though it's likely also not needed there
- Also mention installing the SDK as installing workloads may not get the needed ASP.NET Runtime Packs for 7.0
- Also fixes a typo lol

# Further Context
Follow up to https://github.com/dotnet/msbuild/pull/7485 

@rainersigwald  @edvilme If there were additional things you needed to do to resolve the failures, we should add them to this PR.

cc @Forgind

### CODE REVIEW

File: README.md
Comment (rainersigwald) [inline README.md:46]: This should not be required (the one you get from VS should be sufficient). Did it appear to be?

Comment (nagilson) [reply README.md:46]: Yes, @edvilme encountered some issues even after installing the workloads, and the 4.5. (2?) workloads, he was still facing some runtime pack errors which this fixed

Comment (nagilson) [reply README.md:46]: We might have caused that by fixing one of the earlier errors 🥴 But anyways I'll remove the suggestion, thanks!

**Review by Forgind — APPROVED** [review]

**Comment by nagilson** [comment]
@Forgind  PTAL

**Comment by Forgind** [comment]
LGTM, but we normally wait for at least two sign-offs before merging.

**Review by rainersigwald — APPROVED** [review]

**Pull request merged**
